holmes.py

```diff
@@ -234,11 +244,13 @@ def alertmanager(
         console.print(
             f"[bold yellow]Analyzing all {len(issues)} issues. (Use --alertname to filter.)[/bold yellow] [red]Press Ctrl+C to stop.[/red]"
         )
+    results = []
     for i, issue in enumerate(issues):
         console.print(
             f"[bold yellow]Analyzing issue {i+1}/{len(issues)}: {issue.name}...[/bold yellow]"
         )
         result = ai.investigate(issue, system_prompt, console)
+        results.append({"issue": json.loads(issue.json()), "result": json.loads(result.json())})
```

Comment:
Here too, I don't think the roundtrip is necessary. What are you trying to achive with it?